lib/net.c

```diff
@@ -720,10 +720,62 @@ get_if_back(struct net_config *net_conf)
 	ETH_RSS_NONFRAG_IPV6_OTHER | \
 	ETH_RSS_IPV6_EX)
 
+static int
+check_port_rss_key_update(struct gatekeeper_if *iface, uint16_t port_id,
+	bool *rss_key_updatable, struct rte_eth_rss_conf *rss_conf)
+{
+	struct rte_eth_rss_conf cur_rss_conf;
+	int ret = rte_eth_dev_rss_hash_update(port_id, rss_conf);
```

Comment:
Once `rss_conf` becomes `const` (see another comment), get a local copy of `rss_conf` and pass it to `rte_eth_dev_rss_hash_update()`. It seems that DPDK's developers have forgotten to make the parameter `const`.